packages/manager/modules/pci/src/components/project/instance/instance.class.js

```diff
@@ -206,4 +206,8 @@ export default class Instance {
   isStopped() {
     return this.status === 'SHUTOFF';
   }
+
+  isShelved() {
+    return this.status === 'SHELVED' || this.status === 'SHELVED_OFFLOADED';
```

Comment:
Probably, we can move these statuses to a constant? :)

Reply:
Yes i think too. But as you can see all others function is based on the hard coded status.
So for the moment i will let it, but i planned to create a task to make a refacto.

Reply:
I made a bit changes but i planned to create a task to make the big changes.
Thanks.